org/allPRs.ts

```diff
@@ -224,6 +224,8 @@ export const deploySummary = async () => {
 export const rfc327 = () => {
   const semanticFormat = /^(fix|feat|build|chore|ci|docs|style|refactor|perf|test|revert)(?:\(.+\))?!?:.+$/
 
+  const wipFormat = /^(wip|\[wip\]|do not merge|\[do not merge\]|draft|\[draft\])(?:\(.+\))?!?:.+$/i
```

Comment:
regex is my fav thing, could do something like this to make matching the brackets easier?

```
[\[\(]?[\b($wip)\b($do not merge)\b($draft)][\]\)]?
```

Reply:
not sure if this will work in .ts btw, I think the syntax can be a lil different sometimes

Reply:
it works!